src/Phinx/Console/Command/Create.php

```diff
@@ -131,7 +194,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // Do we need to do the post creation call to the creation class?
+        if (!empty($creationClassName)) {
```

Comment:
I would use `$creationClass` instead of just the name, as it will be operated with below.